Description: If the logfile path doesn't exist, don't attempt to create the log file.  Mainly needed when cobbler is required to run inside the build env (cobbler4j).
Author: Dave Walker (Daviey) <DaveWalker@ubuntu.com>
Forwarded: no
--- a/cobbler/clogger.py
+++ b/cobbler/clogger.py
@@ -39,7 +39,7 @@
       self.logfile = None
 
       # Main logfile is append mode, other logfiles not.
-      if not os.path.exists(logfile):
+      if not os.path.exists(logfile) and os.path.exists(os.path.dirname(logfile)):
          self.logfile = open(logfile, "a")
          self.logfile.close()
 
